### DISCUSSION TITLE
Snake case entity properties names are Camel case in Property Component or Serializer

### DISCUSSION THREAD

**Discussion by Mausino** [symfony/symfony#46358] [Q&A]
I am little bit confusing...
Why entity properties which i have save as snake case are in the Property Component showed like camel case ???

![Snímka obrazovky z 2022-05-15 16-19-22](https://user-images.githubusercontent.com/5773777/168477712-79ef8604-ea97-4839-b0a7-ff379f829ef0.png)
![Snímka obrazovky z 2022-05-15 15-51-29](https://user-images.githubusercontent.com/5773777/168477721-f0a281e0-d45f-4d1d-9080-720ec1e15bcb.png)

Also if I in the serializer attribute write the snake case property.. it doesn'ŧ work.. must write camel case as you see the from debug entity class SystemContacts.

I thinked if i name the property, i would use same property name all accross the symfony.. now i must convert it many times to snake case or camel case...

**Comment by helmis123** [comment]
Try to use the CamelCaseToSnakeCaseNameConverter
`$normalizer = new ObjectNormalizer(null, new CamelCaseToSnakeCaseNameConverter());`

**Reply by darius-v** [reply]
Why it does not use this automatically if it is being used for other entity? 
For example

```
public function getSellerInternalAction(
        #[MapEntity(expr: 'repository.findUndeletedWithChannels(sellerId)')] Seller $seller,
        GetSellerInternalHandler $getSellerInternalHandler
    ): Seller {
        return $getSellerInternalHandler->handle($seller);
    }
```
this controller response has few entties - on one entity they are snake case, on other entity they are camel case, even if using attributes like

`#[SerializedName('dropshipping_type')]`

**Comment by Mausino** [comment]
@helmis123 hi, thank you for hint.. better as my custom code :) but now i thinking... 

it will better convert my entities to camel case in entity class that will no confuse me. only for me it's no reason to have snake case properties if across all application symfony using it as camel case.
